purview/models/distgit.py

```diff
@@ -26,8 +27,9 @@ class DistGitBranch(PurviewStructuredNode):
     repo_name = StringProperty(required=True)
     repo_namespace = StringProperty(required=True)
     commits = RelationshipTo('DistGitCommit', 'CONTAINS')
+    contributors = RelationshipTo('.user.User', 'CONTRIBUTED_BY')
```

Comment:
Any specific reason as to why this relationship is added? I think this wasn't there before right